src/app/components/core/services-health/add-service-wizard/add-service-wizard.component.html

```diff
@@ -0,0 +1,134 @@
+    <div class="step-marker">1</div>
+    <div class="step-details">
+      <p>Service Name &amp; Direction</p>
+    </div>
+  </div>
+  <div class="step-item" id="2">
+    <div class="step-marker">2</div>
+    <div class="step-details">
+      <p>Choose Plugin</p>
+    </div>
+  </div>
+  <div class="step-item" id="3">
+    <div class="step-marker">3</div>
+    <div class="step-details">
+      <p>Review Configuration</p>
+    </div>
+  </div>
+  <div class="step-item" id="4">
+    <div class="step-marker">4</div>
+    <div class="step-details">
+      <p>Start Service</p>
+    </div>
+  </div>
+  <div class="steps-content">
+    <form [formGroup]="serviceForm" validate>
+      <div class="step-content  is-active" id="c-1">
+        <div class="field is-horizontal form-group">
+          <div class="field-label is-normal">
+            <label class="label">Name</label>
+          </div>
+          <div class="field-body">
+            <div class="field">
+              <div class="control">
+                <input class="input" id="name" type="text" placeholder="name" formControlName="name" required  (input)="validateServiceName($event)">
+                <small *ngIf="!isValidName" class="help is-danger level-left">Service name is required</small>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="field is-horizontal form-group">
+          <div class="field-label is-normal">
+            <label class="label">Direction</label>
+          </div>
+          <div class="field-body">
+            <div class="field">
+              <div class="control">
+                <div class="select is-fullwidth">
+                  <select name="type" formControlName="type">
+                    <option value='north'>North</option>
+                    <option value='south'>South</option>
+                  </select>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="step-content has-text-centered" id="c-2">
+        <div class="field is-horizontal form-group">
+          <div class="field-label is-normal">
+            <label class="label">Plugin</label>
+          </div>
+          <div class="field-body">
+            <div class="field">
+              <div class="control">
+                <div class="select is-fullwidth">
+                  <select name="type" formControlName="plugin" (change)="validatePluginValue($event)" required>
+                    <option value='select' disabled>Select</option>
+                    <option *ngFor="let p of plugins" [value]="p.name"> {{ p.name }}</option>
+                  </select>
+                </div>
+                <small *ngIf="!isValidPlugin" class="help is-danger level-left">Please select a valid plugin</small>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="step-content has-text-centered" id="c-3">
```

Comment:
Can we move this to a component and use here and config too? Else it will be a debt to handle at 2 places